terrainnotes.md

```diff
@@ -0,0 +1,113 @@
+To refine past a tile, one of the following conditions must be met:
```

Comment:
Would be great to move these to https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation

Maybe in a new `ImplementationNotes` or whatever subdirectory.

Reply:
Mostly just a brain dump and probably not worth keeping around. But maybe I'll turn the most useful parts into a blog post or something.

Reply:
Blog post would be great!